### DISCUSSION TITLE
Downloaded 1.54 and the version is missing the org.jfree.chart.ui package and the demo files won't run

### DISCUSSION THREAD

**Discussion by jenniferwillowm** [jfree/jfreechart#338] [General]
Hi folks,
     I have been using JFreechart for well over a decade but I haven't updated to the newer code for probably 5 years or more.  I have a new project that requires some chart features that aren't in the version of JFreechart that I've been using.  I decided to update to the newest version (1.54 with 1.52 documentation).  I use Netbeans and I've combined the source tree for JFreechart, OrsonCharts together into my standard class path build tree.  All of the demo programs (jfreechart-demos-1.5.2) are missing the same package (org.jfreechart.chart.ui).  Has this package been deprecated?  
     I originally tried to merge this package from the earlier version I've been running but I decided that just didn't make sense.  However, that means I'm stuck and can't upgrade to the new version.  Could someone explain what's going on?  Has this package been deprecated or moved to another location?
Sincerely,
Jennifer Milburn
jwm@astro.caltech.edu

**Comment by trashgod** [comment]
The demos are now [here](https://github.com/jfree/jfreechart#demos) and you may need to look at issues related to [migration](https://github.com/jfree/jfreechart#migration).

**Comment by jenniferwillowm** [comment]
The issue turned out to be limited to trying to clone the repository.  When I downloaded the tar file it included everything.
